vmm/src/vmm_config/drive.rs

```diff
@@ -158,72 +157,84 @@ impl BlockDeviceConfigs {
     }
 
     fn create(&mut self, block_device_config: BlockDeviceConfig) -> Result<()> {
-        // check if the path exists
+        // If no path exists, then an error is returned.
```

Comment:
nit: ```Check if the path on host is valid``` (or no comment at all as I find the code pretty obvious). 

The comments are for explaining the code, the logic, the reason for doing something and not repeating it in words.